redash/settings/__init__.py

```diff
@@ -36,6 +36,9 @@
 SQLALCHEMY_DISABLE_POOL = parse_boolean(
     os.environ.get("SQLALCHEMY_DISABLE_POOL", "false")
 )
+SQLALCHEMY_ENABLE_POOL_PRE_PING = parse_boolean(
+   os.environ.get("SQLALCHEMY_ENABLE_POOL_PRE_PING", "true")
```

Comment:
```suggestion
   os.environ.get("SQLALCHEMY_ENABLE_POOL_PRE_PING", "false")
```

Default value should be false to keep current behavior.